swarm/network/protocol.go

```diff
@@ -209,7 +212,11 @@ func (b *Bzz) RunProtocol(spec *protocols.Spec, run func(*BzzPeer) error) func(*
 			localAddr:  b.localAddr,
 			BzzAddr:    handshake.peerAddr,
 			lastActive: time.Now(),
+			LightNode:  handshake.LightNode,
 		}
+
+		log.Info("peer created: %s", handshake.peerAddr.String())
```

Comment:
I think this should be a `log.Debug`, otherwise logs will be full of it. Also use structured logs:

```
log.Debug("peer created", "addr", handshake.peerAddr.String())
```

Reply:
Fixed logging